### PR TITLE
AboghanemAO_lab_1

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: AboghanemAO/lab1/Source.cpp [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+            cin >> operation_sign;
+
+
+
+            switch (operation_sign)
+            {
+
+            case '+':
+                std::cout << "Your answer is : " << a + b << endl;
```
Comment (Toxa-man) [inline AboghanemAO/lab1/Source.cpp]: Please extract line 
`std::cout << "Your answer is : " << a ? b << endl;` from switch..case in order to reduce code duplication

Comment (ghost) [reply AboghanemAO/lab1/Source.cpp]: fixed
but i have a question :
now when the operation sign is wrong the output will be
"Your answer is : Wrong operation sign try again"
should i add (if statement) before switch ? so if the the operation sign is wrong it doesn't print "Your answer is :"

File: AboghanemAO/lab1/Source.cpp [hunk 2]
```diff
@@ -0,0 +1,79 @@
+#include <iostream>
+#include <conio.h>
+using namespace std;
+
+
+
+int main() {
+
+    float a;
```
Comment (Toxa-man) [inline AboghanemAO/lab1/Source.cpp]: Give meaningful names to variables

**Review by Toxa-man — COMMENTED** [review]
Code issues
Wrong folder name

File: AboghanemAO/lab1/Source.cpp (hunk 1 above)
Comment (Toxa-man) [reply AboghanemAO/lab1/Source.cpp]: Yes, it is the proper code flow

**Pull request closed without merging**